diff --git a/node_modules/react-native-fast-tflite/react-native-fast-tflite.podspec b/node_modules/react-native-fast-tflite/react-native-fast-tflite.podspec
index 2a4d852..e2d21ae 100644
--- a/node_modules/react-native-fast-tflite/react-native-fast-tflite.podspec
+++ b/node_modules/react-native-fast-tflite/react-native-fast-tflite.podspec
@@ -23,6 +23,7 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm}", "cpp/**/*.{hpp,cpp,c,h}"
 
   s.pod_target_xcconfig = {
+    'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64 i386',
     'GCC_PREPROCESSOR_DEFINITIONS' => "$(inherited) FAST_TFLITE_ENABLE_CORE_ML=#{enableCoreMLDelegate}",
     'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
   }
