diff --git a/node_modules/vision-camera-resize-plugin/ios/FrameBuffer.mm b/node_modules/vision-camera-resize-plugin/ios/FrameBuffer.mm
index 5fab069..3805fc8 100644
--- a/node_modules/vision-camera-resize-plugin/ios/FrameBuffer.mm
+++ b/node_modules/vision-camera-resize-plugin/ios/FrameBuffer.mm
@@ -7,6 +7,7 @@
 //
 
 #import "FrameBuffer.h"
+#import <stdint.h>
 
 @implementation FrameBuffer {
   vImage_Buffer _imageBuffer;
